From 6aeea1a7abd05745fab666b8fadab94caed6d53a Mon Sep 17 00:00:00 2001
From: Julien Cristau <jcristau@debian.org>
Date: Mon, 4 Aug 2008 12:18:12 +0200
Subject: [PATCH] Fix up the HP Pavilion ze4944ea quirk

The chip is 855GM, not GM45.
(cherry picked from commit 1a59cc6b9acf312de1755d67757bf7f1967342e4)

diff --git a/src/i830_quirks.c b/src/i830_quirks.c
index 5ae2898..6fc8e53 100644
--- a/src/i830_quirks.c
+++ b/src/i830_quirks.c
@@ -271,8 +271,6 @@ static i830_quirk i830_quirk_list[] = {
 
     /* HP Compaq 6730s has no TV output */
     { PCI_CHIP_GM45_GM, 0x103c, 0x30e8, quirk_ignore_tv },
-    /* HP Pavilion ze4944ea needs pipe A force quirk (See LP: #242389) */
-    { PCI_CHIP_GM45_GM, 0x103c, 0x3084, quirk_pipea_force },
 
     /* Thinkpad R31 needs pipe A force quirk */
     { PCI_CHIP_I830_M, 0x1014, 0x0505, quirk_pipea_force },
@@ -292,6 +290,8 @@ static i830_quirk i830_quirk_list[] = {
     { PCI_CHIP_I855_GM, 0x1028, 0x00c8, quirk_pipea_force },
     /* Intel 855GM hardware (See Novell Bugzilla #406123) */
     { PCI_CHIP_I855_GM, 0x10cf, 0x1215, quirk_pipea_force },
+    /* HP Pavilion ze4944ea needs pipe A force quirk (See LP: #242389) */
+    { PCI_CHIP_I855_GM, 0x103c, 0x3084, quirk_pipea_force },
 
     /* ThinkPad X40 needs pipe A force quirk */
     { PCI_CHIP_I855_GM, 0x1014, 0x0557, quirk_pipea_force },
-- 
1.5.6.4

